modules/01-mods.js

```diff
@@ -344,7 +344,7 @@ async function slashModSlowmode(interaction) {
 async function slashModSummary(interaction) {
   await interaction.deferReply({ ephemeral: true });
   const member = interaction.options.getMember("user");
-  const time = interaction.options.get("history").value ?? 28;
+  const time = interaction.options.get("history") ? interaction.options.get("history").value : 28;
```

Comment:
no need for the terniary.
object?.value essentially means
```
if (object) return object.value
else return undefined
```
when mixed with a ?? which sets another value if the one on the left is nullish, we can get a second option. Also I'm pretty sure options.get is outdated. 

```suggestion
  const time = interaction.options.getInteger("history")?.value ?? 28;
```